### DISCUSSION TITLE
Rentability with 72TB (about $ 3000 of investment)

### DISCUSSION THREAD

**Discussion by Freenitial** [Chia-Network/chia-blockchain#3056] [Q&A]
Excuse me guys, there is something that I am not sure to understand. And sorry if english is bad i use translator.

I did a simulation over 5 years with the basic parameters, and the curves indicate that if tommorow I invest in 72TB of HDD, after a year I will have earned $ 1350 then after nothing more in the scenario of the chia at $ 20 and network grow at 6%.

If I understood correctly, the most probable scenarios are sad for individuals like me, yet I have the feeling that investing $ 3000 is already a lot but in fact this currency is expected to be really profitable only for very rich investors, it would force me to constantly reinvest my earnings in new HDDs to keep pace with the total network space to keep earning.

It's a hellish loop, to really make money there are only 2 ways: either you invest tens of thousands of dollars or you pray that the chia is worth $1000.

I really hope you will answer me please because every minute that passes increases my risk of getting back on the train (if I understood correctly).
I wish I was wrong and had at least a 50/50 chance of paying back my investment, but I feel that with $ 3000 it's not even worth thinking about.
I'm just a novice but I don't have time to spend weeks learning about the odds because it will be too late to start after that. I am sure that one of you reading this is able to give me a clearer idea of ​​the risks I am taking, and I will be grateful to you.

I found someone on github who asked pretty much the same question, and the only answer was, "don't worry there will be pools soon" except that doesn't change the problem; if I earn $ 1350 in a year then 3 cents per months, it will not reimburse my investment.
I am surprised that this subject is so little discussed when the project has been evolving for months and the developers are necessarily aware of this yet I haven't found anything official about it to warn the poor people who buy a few hundred dollars worth of HDD that they're throwing their money out the window, it's not very nice for them.
Obviously this kind of investment is risky, but in my current state of understanding the context, it is sheer madness!

In case my understanding of the context is correct, a warning should be written in large letters everywhere, on the home page of the wiki, when you launch the software etc because I myself, having searched for several hours, cannot find a clear answer to this problem... as if it was absolutely obvious and that everyone had understood... yet I see small farmers with a few TBs who seem serene.
Which again leads me to wonder if it is they who are wrong to farm or me who misunderstood something. So again, I hope I'm wrong and that farming can be profitable for small investors.

sorry for this indigestible and repetitive text, but I prefer that the question be perfectly clear so that I have a chance to have a clear answer as well

SO MY QUESTION IS : Am I wrong or finally chia is not so interesting for a 70 TB farm ???

**Comment by SackofFluor** [comment]
It just isnt....

If you are not having "by chance" a good access to harddrives and also computational power to do the plotting you cannot consider it an investment.

The "average" person is not going to get any of their invested money back if a single chia ist going through the roof once they become tradable...

**Reply by Freenitial** [reply]
Thank you for your answer

**Comment by grinbucks** [comment]
A very good point to be discussed.
IMHO, calculating the return at this stage is almost useless due to the limited data available.
However, we could calculate different scenarios by changing the variables (e.g. price, network growth, etc).
We also should consider that investing in crypto is so much different than investing in other asset classes.
There are many cases where buying the coins seems more logical than investing in the mining rigs.
As for the 70TB farm, at least you know one thing for sure, your downside is limited to the purchase cost of the rigs and the running cost (the big chunk of it is electricity, easily calculated). The upside potential is pretty much unlimited if you hodl the coin.

**Comment by pluto2021** [comment]
It is a risky business at this point. As an old saying says: investment carries risk, and invest only what you can lose. Without a pool, this Chia race is not an investment at all, it feels like a game with the house already played and earned a huge portion. It is a lottery with an unknown price tag. So you cannot even calculate the return on "investment". I would highly doubt the price accuracy at the Chiacalculator.com, considering the probability of winning vs. the amount of $$$ putting in.  The price could soar if the founders would not sell what they have in hand (like an IPO, except that there is currently no regulations on how long the founders need to hold their shares, in this case, the amount of CHX prefarmed). If they decide to sell, the price could tank quickly, while the founders get their great efforts paid sooner, it would make continuous investment in chia space growth almost nonsense. Eventually, this would lead to very high volatility of the price at first. slowly getting into some sort of balance.  I could imaging that there would be huge amount of used HDDs for sale on ebay when CHX's price would tank.

Even though the network space is growing at an exponential rate with astronomical ceiling, the rewarding system is only at the beginning stage. Hope that with so much efforts put into the plotting, along with that HUGE energy bill behind, there would some kind of algorithm making the Chia blockchain ecosystem more sustainable.

So, if your life depends on this $3000 to pay your everyday bills, putting that amount into Chia might not be a good idea. Theoretically, people could abandon this blockchain and the price would approach zero at some point. Then you would just lose that $3000, plus all the psychological pain, and the quality of life, etc.

BTW, the syncing is extremely slow somehow. So even if you manage to get the plotting done right, the syncing goes linearly, while the netspace is growing exponentially. So eventually, at some point one would never be able to sync. Without syncing, your plots may not even get entered to the lottery draw.
